docs/features/dynamic-analysis.work.md

```diff
@@ -0,0 +1,14 @@
+Dynamic Analysis
+===============
+
+This feature enables instrumentation of binaries to collect information for dynamic analysis.
+
+--------------------
+
+TODO:
+
+- Generate code to compute MVIDs less often than once per invoked method.
```

Comment:
See this [example](https://github.com/gafter/roslyn/blob/3c0b3fe0d3eba565b7884908063c68ffec92c090/docs/features/records.md).
Haven't tried it yet, but I think it's two square brackets with a space or an x in the middle. [ ] or [x]